backend/tests/integration/test_b07_p4_operational_readiness_e2e.py

```
@@ -0,0 +1,578 @@
+        "id": str(tenant_id),
+        "name": f"B07 P4 Tenant {tenant_id.hex[:8]}",
+        "api_key_hash": f"hash_{tenant_id.hex[:16]}",
+        "notification_email": f"{tenant_id.hex[:8]}@test.invalid",
+        "shopify_webhook_secret": "test_secret",
+        "created_at": now,
+        "updated_at": now,
+    }
+    insert_cols = []
+    for col in required:
+        if col not in payload:
+            raise RuntimeError(f"Missing payload value for required tenants column '{col}'")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)